src/qtoolkit/io/sge.py

```diff
@@ -280,6 +280,10 @@ def _get_qstat_base_command(self) -> list[str]:
     def _get_job_ids_flag(self, job_ids_str: str) -> str:
         return f"-j {job_ids_str}"
 
+    def _get_job_cmd(self, job_id: str):
+        cmd = f"qstat -j {job_id}"
+        return cmd
```

Comment:
In case of SGE, I'm not sure if I understood it correctly if we always need the `-u` option (instead of `-j`)?
Otherwise, also here we could parse this as an xml file maybe? 
```suggestion
    def _get_job_cmd(self, job_id: str):
        cmd = f"qstat -j {job_id} -xml"
        return cmd
```

Reply:
Otherwise, I will switch this to the -u option.

Reply:
I think that the `-xml` option can indeed be moved to  `_get_job_cmd` for SGE